Tests/Decoda/Hook/EmoticonHookTest.php

```diff
@@ -5,20 +5,11 @@
 use FM\BbcodeBundle\Emoticon\Emoticon;
 use FM\BbcodeBundle\Emoticon\EmoticonCollection;
 use FM\BbcodeBundle\Decoda\Hook\EmoticonHook;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use FM\BbcodeBundle\Emoticon\Loader\FileLoader;
 
-class EmoticonHookTest extends \PHPUnit_Framework_TestCase
+class EmoticonHookTest extends \PHPUnit\Framework\TestCase
 {
-    public function testConstructor()
-    {
-        $loader = $this->getMockBuilder('FM\\BbcodeBundle\\Emoticon\\Loader\\FileLoader')
-            ->disableOriginalConstructor()
-            ->getMockForAbstractClass()
-        ;
-        $container = $this->getMock('Symfony\\Component\\DependencyInjection\\ContainerInterface');
-
-        $result = new EmoticonHook($loader, $container);
-    }
-
```

Comment:
What is the reason to delete this case?

The assertion here is that no exception is thrown.

Reply:
Idem.